Content.Server/Chemistry/ReagentEffects/RemoveReplacementAccent.cs

```diff
@@ -0,0 +1,30 @@
+using Content.Server.Speech.Components;
+
+using Content.Shared.Chemistry.Reagent;
+
+namespace Content.Server.Chemistry.ReagentEffects;
+
+/// <summary>
+/// Removes the ReplacementAccentComponent on the target, if any.
+/// </summary>
+public sealed class RemoveReplacementAccent : ReagentEffect
```

Comment:
should have access

Reply:
I appreciate the review, but I don't understand. As far as I can tell from google, this class already has maximum access from the public class, and aside from that every other chemical effect shares these same modifiers. Could you elaborate?

Reply:
`[Access(typeof(WhateverSystem))]`
prevents classes that dont have access from modifying it